src/reducers/index.js

```diff
@@ -0,0 +1,34 @@
+const mockUpBooks = [
+  {
+    id: Math.floor(Math.random() * 100),
+    title: 'The old man and the sea',
+    category: 'Kids',
+
+  },
+  {
+    id: Math.floor(Math.random() * 100),
+    title: 'The knight in the rusty armor',
+    category: 'Action',
+
+  },
+  {
+    id: Math.floor(Math.random() * 100),
+    title: 'Alive',
+    category: 'Action',
+
+  },
+  {
+    id: Math.floor(Math.random() * 100),
+    title: 'The Butterfly Effect',
+    category: 'Sci-Fi',
+
+  },
+  {
+    id: Math.floor(Math.random() * 100),
+    title: 'Secret weapons of World War II',
+    category: 'History',
+
+  },
+];
+
+export default mockUpBooks;
```

Comment:
- [ ] The index.js file at the reducer directory should combine all reducers and export them. Since now we only have the book reducer, the file should only export it.
```JS
import { combineReducers } from 'redux'
import books from './books'

export default combineReducers({
  books
})
```

- [ ] The mockUpBooks should be placed at the book.js reducer file.